packages/react-native-editor/__device-tests__/slash-inserter.test.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Internal dependencies
+ */
+import { blockNames } from './pages/editor-page';
+import { isAndroid } from './helpers/utils';
+
+describe( 'Gutenberg Editor tests for slash inserter', () => {
+	it( 'should insert an image block with enter', async () => {
```

Comment:
Can we be more specific here. 
```suggestion
	it( 'should insert an image block with "/img" + enter', async () => {
```